.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "CAPA Devcontainer + Devbox",
+  "image": "mcr.microsoft.com/devcontainers/base",
+  "features": {
+    "ghcr.io/dlouwers/devcontainer-features/devbox:1": {},
+    "ghcr.io/devcontainers/features/docker-in-docker:2.12.0": {},
+    "ghcr.io/audacioustux/devcontainers/tilt:1.0.1": {},
```

Comment:
Tilt is not needed in the devcontainer as its specified to the devbox.

Reply:
The nix package for tilt seems to be broken, I'll test to confirm the exact error.

Reply:
Oh thats not good, would love to know why its not working.